ra2ce/analysis/damages/damage_functions/damage_fraction_uniform.py

```diff
@@ -39,20 +39,23 @@ class DamageFractionUniform:
     self.data (pd.DataFrame) : index = hazard severity (e.g. flood depth); column 0 = damage fraction
     """
 
-    name: str = None
-    hazard_unit: str = None
-    data: pd.DataFrame = None
-    origin_path: Path = None
+    name: str
+    hazard_unit: str
+    data: pd.DataFrame
+    origin_path: Path
     interpolator: interp1d = None
 
+    def __post_init__(self):
+        self._convert_hazard_severity_unit("m")
+
     @classmethod
-    def from_csv(cls, path: Path, sep=",") -> DamageFractionUniform:
+    def from_csv(cls, csv_path: Path, sep: str) -> DamageFractionUniform:
         """Construct object from csv file. Damage curve name is inferred from filename
 
         Arguments:
-            *path* (Path) : Path to the csv file
+            *csv_path* (Path) : Path to the csv file
             *sep* (str) : csv seperator
-            *output_unit* (str) : desired output unit (default = 'm')
+            *output_unit* (str) : desired output unit
```

Comment:
This is no longer an argument, is it? Eitherway, these are not the docstrings format we use but an outdated one.

Reply:
Done